arch/arm/soc/nordic_nrf5/nrf52/dts.fixup

```diff
@@ -1,7 +1,10 @@
+/* SoC level DTS fixup file */
+
 #define CONFIG_NUM_IRQ_PRIO_BITS	ARM_V7M_NVIC_E000E100_ARM_NUM_IRQ_PRIORITY_BITS
 #define CONFIG_UART_NRF5_IRQ_PRI	NORDIC_NRF_UART_40002000_IRQ_0_PRIORITY
 #define CONFIG_UART_NRF5_BAUD_RATE	NORDIC_NRF_UART_40002000_CURRENT_SPEED
 #define CONFIG_UART_NRF5_NAME		NORDIC_NRF_UART_40002000_LABEL
 
-#define CONFIG_SOC_FLASH_NRF5_DEV_NAME	NRF_NRF52_FLASH_CONTROLLER_4001E000_LABEL
-#define FLASH_DEV_NAME			CONFIG_SOC_FLASH_NRF5_DEV_NAME
+#define FLASH_DEV_NAME			NRF_NRF52_FLASH_CONTROLLER_4001E000_LABEL
```

Comment:
Should we remove it from all  the soc.h files then?

Reply:
Remove what from the soc.h files?

Reply:
Ah sorry, we define `FLASH_DRIVER_NAME` to `FLASH_DEV_NAME` there, confusing :|

Reply:
Yeah, ideally we'd update mcuboot to use the dt now and remove the need for a define.